frontend/src/services/userAuthService.ts

```diff
@@ -14,44 +15,56 @@ export interface UserCredentials {
     password: string;
 }
 
-export const submitUserData = async (userData: UserData) => {
-    try {
-        const response = await axios.post('http://34.172.117.230:8080/api/v1/account', userData);
+export const userAuthService = {
+    register: async (userData: UserData) => {
+        try {
+            const response = await axios.post('/v1/account', userData);
+            const { accessToken } = response.data;
+
+            if (!accessToken) {
```

Comment:
Z tego co widzę (ale to jeszcze sprwawdź) przy rejestracji backend nie zwraca accessTokena, więc teraz za każdym razem będzie ci rzucało błędem. Zamiast tego możesz spojrzeć jaki dostajesz response code i jeśli nie jest to 201 to wtedy rzucić błędem